src/js/components/LogItem/utils/decimalToHex.tsx

```diff
@@ -0,0 +1,3 @@
+export const decimalToHex = (decimal: number, prefix = '0x') => (
```

Comment:
how about use it from jooby-codec/utils?

Reply:
discussed

for now, we decided to leave it as it is